extension/src/plots/model/custom.ts

```diff
@@ -0,0 +1,25 @@
+import { CheckpointPlot, CustomPlot, CustomPlotType } from '../webview/contract'
+
+export const CHECKPOINTS_PARAM = 'epoch'
+
+export type CustomPlotsOrderValue = {
+  type: CustomPlotType.METRIC_VS_PARAM | CustomPlotType.CHECKPOINT
+  metric: string
+  param: string
+}
+
+export const isCheckpointValue = (
```

Comment:
[I] Most of the usage of this can be replaced with `isCheckpointPlot`. E.g 

```
  const plotsItems = plots.map(plot =>
    isCheckpointValue(plot.type)
      ? getCheckpointPlotItem(plot.metric)
      : getMetricVsParamPlotItem(plot.metric, plot.param)
  )
```

```
      param: isCheckpointValue(plot.type)
        ? plot.param
        : buildMetricOrParamPath(ColumnType.PARAMS, plot.param),
```

Reply:
`isCheckpointPlot` wouldn't work for those instances since they're inspecting a order item, not a `CheckpointPlot` 🤔

Reply:
😕. Ok, the problem is that `plots` are not `plots` they `CustomPlotsOrderValue`s. The variable name should reflect what they are.

<img width="559" alt="image" src="https://user-images.githubusercontent.com/37993418/224859359-160be343-f097-4b06-8d8c-85b30197d843.png">

Reply:
Oh, yes, you're right! I need to take a break, my brain is not thinking straight 😅